portfolio/src/main/webapp/copyright.js

```diff
@@ -0,0 +1,2 @@
+// This displays the current year of the copyright.
+document.getElementById("year").innerHTML = new Date().getFullYear();
```

Comment:
While this is completely fine for your project, one thing to think about when you work on larger projects is that the id that you choose needs to be unique within the entire loaded page. Here, something like "copyright-year" might be a bit less likely to conflict and be easier for someone to search your project for if it ever needs to be updated.

Reply:
Gotcha, I will work on it.